brainstormforce/astra
# AST-653 - Added compatibility for search block in WordPress 5.8

### Description
- Added compatibility for search block introducing in WP-5.8
- As this block is in WP-5.7 versions as well, added backward for old users to not affect anything directly

### Screenshots
- with master version - https://share.bsf.io/nOuP7OrG
- with this branch - https://share.bsf.io/v1uY9kkx

### Types of changes
- Added GB editor + Frontend support for search block

### How has this been tested?
- Check with different search box styles (search block)
- Check on all devices
- Check GB editor view
- Button styles should work for search buttons
- Check search block button with text/icons
- Check different search button styles provided in the block

### Checklist:
